modules/Settings/Colors/views/PickerModal.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Base color picker modal view class.
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Tomasz Poradzewski <t.poradzewski@yetiforce.com>
+ */
+class Settings_Colors_PickerModal_View extends \App\Controller\ModalSettings
```

Comment:
ModalSettings?
We should have the same here.
![image](https://user-images.githubusercontent.com/10221999/70962681-44312780-2086-11ea-9c34-1465d3027d79.png)